db/seeds.rb

```diff
@@ -5,3 +5,14 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+Provider.destroy_all
```

Comment:
Fixed in https://github.com/tpitale/counselor/commit/b47e8661555bdb751706ae7e772fbda325c05049